### DISCUSSION TITLE
[Question][Hw0-2] Testing code not working

### DISCUSSION THREAD

**Discussion by 727yubin** [prosyslab-classroom/cs348-information-security#645] [Q&A]
Name: Yubin Lee

In HW0-2, in the "Testing Your Code" section, the `dune exec test/[Problem name]Test.exe` command is given.
However, when I try to execute this command, I get the following error:

`$ dune exec test/zipTest.ml 
Error:                          
execve(/home/student/cs348-2023-spring-hw0-2-_________/_build/default/test/zipTest.ml): Permission denied`

This happens both in the KCLOUD VM and the Docker image.

How can I fix this? I have ownership of all the files...

**Comment by bongjunj** [comment]
It usually happens when you run `dune exec test/[Problem name]Test.ml` rather than `dune exec test/[Problem name]Test.exe`. Look at the different file extensions.

**Comment by 727yubin** [comment]
Oh I didn't read it properly in the documentation.. how stupid of me... thank you..!!!
